### DISCUSSION TITLE
Start harbor with a different base or root url

### DISCUSSION THREAD

**Discussion by amrap030** [goharbor/harbor#19361] [Q&A]
Hello together,

I installed harbor in my Kubernetes Cluster behind a traefik reverse proxy via the path `https://<hostname>/harbor`. When I try to access the application, the first request is fine, however, all subsequent requests are being made to `https://<hostname>` without the path `/harbor`.

How can I start the application with a different base or root url, such that every request from the frontend application is being made against `https://<hostname>/harbor`? Otherwise I am not able to route correctly.

Thank you all in advance!

**Comment by danielzhanghl** [comment]
I tried some time ago, and found its client behavior, like Docker CLI, they always try to access root context for the authorization. 
you have to use another domain to separate the traffic with another application.
